src/Infrastructure/AmountParser.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Infrastructure;
+
+/**
+ * @licence GNU GPL v2+
+ * @author Kai Nissen < kai.nissen@wikimedia.de >
+ * @author Jeroen De Dauw < jeroendedauw@gmail.com >
+ */
+class AmountParser {
+
+	private $locale;
+
+	public function __construct( string $locale = 'de_DE' ) {
```

Comment:
Make `$locale` required maybe? There are no cases in which `AmountParser` is created without passing this parameter, anyway.